docs/fundamentals/code-analysis/style-rules/naming-rules.md

```diff
@@ -19,31 +19,29 @@ helpviewer_keywords:
 ---
 # Naming rules
 
-Naming rules concern the naming of .NET programming language code elements, such as classes, properties, and methods. For example, you can specify that public members must be capitalized or that private fields must begin with `_`.
+In your `.editorconfig` file, you can define **naming rules** for how .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;should be named. For example, you can specify that public members must be capitalized, or that private fields must begin with `_`.
```

Comment:
```suggestion
In your `.editorconfig` file, you can define *naming rules* for how .NET programming language code elements&mdash;such as classes, properties, and methods&mdash;should be named. For example, you can specify that public members must be capitalized, or that private fields must begin with `_`.
```

Reply:
The emphasis is trying to get across that these three element kinds can/are defined in the EditorConfig file: naming rules, symbol groups, and naming styles. But it feels repetitious to say that explicitly, and then to say "a naming rule is composed of:" followed by a second bullet list.

I don't feel italics is strong enough for that purpose.